code/parachain/frame/composable-traits/src/staking/mod.rs

```diff
@@ -54,6 +54,19 @@ impl<Balance> RewardRate<Balance> {
 		Self { period: RewardRatePeriod::PerSecond, amount: amount.into() }
 	}
 }
+impl<Balance> RewardRate<Balance>
+where
+	Balance: Copy + Into<u128> + From<u128>,
+{
+	/// The amount that will be rewarded in one period.
+	// TODO(benluelo): Encode somehow that one period will always be rewardable. We can most likely
```

Comment:
We need to consider not adding so many TODOs and may be use something like `IMPROVEMENT`. TODO implies a definite thing to do. Otherwise code looks incomplete to the reader, which is not the case here.